README.md

```diff
@@ -1,2 +1,37 @@
-# intuit-login-page-clone
-This is a clone of intuit website's login page clone
+# intuit-signup-page-clone
+This is a clone of intuit website's signup page clone
+
+## BUILT WITH
+
+- HTML
+
+- CSS
```

Comment:
[Optional]

We can still list the features of HTML/CSS we used, for example, CSS flexbox or HTML forms.

Reply:
Thanks for the comment. I'll add that